From 540e1b1ebc3718480340c82f114f64bb5987dcc8 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.xyz>
Date: Fri, 2 Dec 2016 23:13:44 +0800
Subject: [PATCH 2/2] arm: dts: sun8i: reuse the uart1 node of iNet D978 rev2
 board

As a uart1 node is added into sun8i-reference-design-tablet.dtsi, simply
use it in iNet D978 rev2 device tree.

Signed-off-by: Icenowy Zheng <icenowy@aosc.xyz>
---
 arch/arm/boot/dts/sun8i-a33-inet-d978-rev2.dts | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/arch/arm/boot/dts/sun8i-a33-inet-d978-rev2.dts b/arch/arm/boot/dts/sun8i-a33-inet-d978-rev2.dts
index fb46655..7335461 100644
--- a/arch/arm/boot/dts/sun8i-a33-inet-d978-rev2.dts
+++ b/arch/arm/boot/dts/sun8i-a33-inet-d978-rev2.dts
@@ -49,15 +49,6 @@
 	model = "INet-D978 Rev 02";
 	compatible = "primux,inet-d978-rev2", "allwinner,sun8i-a33";
 
-	aliases {
-		serial0 = &uart1;
-	};
-
-	chosen {
-		/* Delete debug UART as serial0 is the UART for bluetooth */
-		/delete-property/stdout-path;
-	};
-
 	leds {
 		compatible = "gpio-leds";
 		pinctrl-names = "default";
@@ -96,13 +87,6 @@
 	};
 };
 
-&r_uart {
-	status = "disabled";
-};
-
 &uart1 {
-	pinctrl-names = "default";
-	pinctrl-0 = <&uart1_pins_a>,
-		    <&uart1_pins_cts_rts_a>;
 	status = "okay";
 };
-- 
2.10.2

